packages/react-mutation-mapper/src/component/column/Signal.tsx

```diff
@@ -0,0 +1,147 @@
+import * as React from 'react';
+import 'rc-tooltip/assets/bootstrap_white.css';
+import {
+    extendMutations,
+    getVariantAnnotation,
+    IExtendedSignalMutation,
+    Mutation,
+    RemoteData,
+    formatNumberValueInSignificantDigits,
+    defaultSortMethod,
+} from 'cbioportal-utils';
+import { VariantAnnotation } from 'genome-nexus-ts-api-client';
+import {
+    MutationTumorTypeFrequencyTable,
+    FrequencyTableColumnEnum,
+    DefaultTooltip,
+    FREQUENCY_COLUMNS_DEFINITION,
+} from 'cbioportal-frontend-commons';
+import { observer } from 'mobx-react';
+import { errorIcon, loaderIcon } from '../StatusHelpers';
+type SignalProps = {
+    mutation: Mutation;
+    indexedVariantAnnotations?: RemoteData<
+        { [genomicLocation: string]: VariantAnnotation } | undefined
+    >;
+};
+
+export function getSignalcColumnData(
```

Comment:
There is a typo here, right?

```suggestion
export function getSignalColumnData(
```

We can maybe just name it like this:

```suggestion
export function getSignalData(
```